public/app/features/canvas/elements/parallelogram.tsx

```diff
@@ -0,0 +1,170 @@
+import { css } from '@emotion/css';
+import React, { PureComponent } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { config } from 'app/core/config';
+import { DimensionContext } from 'app/features/dimensions';
+import { ColorDimensionEditor } from 'app/features/dimensions/editors/ColorDimensionEditor';
+import { TextDimensionEditor } from 'app/features/dimensions/editors/TextDimensionEditor';
+import { getDataLinks } from 'app/plugins/panel/canvas/utils';
+
+import { CanvasElementItem, CanvasElementProps, defaultBgColor, defaultTextColor } from '../element';
+import { Align, TextConfig, TextData, VAlign } from '../types';
+
+class Parallelogram extends PureComponent<CanvasElementProps<TextConfig, TextData>> {
+  render() {
+    const { data } = this.props;
+    const styles = getStyles(config.theme2, data);
+
+    return (
+      <div className={styles.container}>
+        <svg
```

Comment:
Any reason why parrellogram / cloud have default borders set already (the rest of elements in canvas don't do this 🤔)

![Screenshot 2024-03-13 at 9 30 40 AM](https://github.com/grafana/grafana/assets/22381771/58ea139f-efbf-49c5-ad16-f42a43f91760)